### DISCUSSION TITLE
FVT/GE Webinar?

### DISCUSSION THREAD

**Discussion by GarnesDoug** [GarnesDoug/FVT-Baylor-University#2] [General]
Hi!

Would anyone be interested in us holding a webinar on how to deploy this?

Thank you,
Doug Garnes

**Comment by MSweebe** [comment]
That would be really helpful! Were looking to use the method you and your team came up with but were having some bumps. One is that were not quite sure what the Student Cohort File is from NSC / where to find that. 

Matt S
